plugin/src/main/java/io/jenkins/plugins/casc/impl/configurators/HeteroDescribableConfigurator.java

```diff
@@ -118,15 +118,18 @@ public CNode describe(T instance, ConfigurationContext context) {
     }
 
     @CheckForNull
-    public void describeStructure(T instance, ConfigurationContext context) {
+    public String describeStructure(T instance, ConfigurationContext context) {
         lookupConfigurator(context, instance.getClass())
             .map(configurator -> convertToNode(context, configurator, instance))
             .filter(Objects::nonNull)
             .map(node -> {
-                    return new Scalar(preferredSymbol(instance.getDescriptor()));
+                if(node.getClass().isEnum()){
+                    return instance.getDescriptor().getDisplayName();
```

Comment:
@casz So if the node is an enumeration probably I could get the descriptors for the node. But as in the jelly we need all of the implementors from the heterodescribable configurators which if we use the instance we cannot seem to obtain.
This would just return all of the descriptors for the heterodescribable configurators right?